registry-packages.ts

```diff
@@ -115,4 +115,14 @@ export const packages: PackageInfo[] = [
     },
     aliases: ['fujing'],
   },
+  {
+    name: '連加連乘',
+    repo: 'XingZiLong/continuous-operation',
+    description: 'Do continuous addition and mutiplication easily.',
+    author: {
+      name: 'XingZiLong',
+      url: 'httos://github.com/XingZiLong',
+    },
+    aliases: ['XingZiLong'],
```

Comment:
Aliases are for package name (normally it's pinyin but English is ok too). Looks like you mistakenly put the username here.

Reply:
I have misunderstood it. Sorry, I will change it.